pkgs/applications/misc/foxtrotgps/default.nix

```diff
@@ -43,6 +43,11 @@ in stdenv.mkDerivation rec {
   chmod -R u+w $sourceRoot/doc/screenshots
   '';
 
+  # Remove when foxtrotgps supports gpsd 3.23.1
+  prePatch = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Changed patching strategy entirely for marble (fetching from upstream git), and changed the foxtrotgps one to postPatch.